Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -374,17 +373,29 @@ final class DeclarationTests: XCTestCase {
       private(1️⃣get, set) var a = 0
       """,
       diagnostics: [
-        DiagnosticSpec(message: "unexpected code 'get,' in modifier")
-      ]
+        DiagnosticSpec(
+          message: "expected 'set' in modifier",
+          fixIts: ["remove 'get,'"]
+        )
+      ],
```

Comment:
If we continue if `detail.detail.presence == .present` then we get `unexpected code 'get,' in modifier` and there is no `"remove 'get,'"` Fix-It